src/create_disk.sh

```diff
@@ -8,27 +8,65 @@
 # although see also https://github.com/coreos/coreos-assembler/pull/298
 # For people building "derived"/custom FCOS-like systems, feel free to file
 # an issue and we can discuss configuration needs.
-set -euo pipefail
```

Comment:
Was this accidental?

Reply:
Nope, it got _moved_.  Otherwise having `set -eu` fails when you try to shift the cli arguments and there are no more to shift.

Reply:
You can `shift` ok _once_ after you're out of args I think (that's what the old code does). Could the kargs code be shifting too much since they're all passed in as one string?

Reply:
Yeah, its my "too-clever for its own good" arg parsing for karg. I dropped that and pushed a cleaner version.